BUILDING.md

```diff
@@ -12,8 +12,127 @@ For building the manual, see the README in the [Manual repository](https://githu
 
 Windows
 -------
+ Visual studio version:
+ 1.) Download:
+   https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017
+
+   Install: Tools for Visual Studio 2019
+     MSVC v142 
+     Windows 10 SDK
+     C++ Cmake Tools for Windows
+     MSVC v141 - VS2017 C++ x64/x86 build Tools
+
+ 2.) Download:
+   http://download.qt.io/official_releases/qt/5.9/5.9.7/qt-opensource-windows-x86-5.9.7.exe
+   Install 
+      MSVC 2017 64-bit
+ 3.) Compile (Console)
+
+   set Qt5_DIR=E:\Qt\Qt5.9.7\5.9.7\msvc2017_64\lib\cmake\Qt5
```

Comment:
I would pass -DCMAKE_PREFIX_PATH=E:\Qt\Qt5.9.7\5.9.7\msvc2017_64 to cmake instead. Also, you need to explain the path: some user may copy the line exactly while the installation path is not the same.

Reply:
Thanks for looking into my changes. This was a recommendation somewhere on a website, howto compile with Qt5 on windows. But I wanted to try our the VS build again and hopefully find a fix for the crash, that 'm currently seeing. I will try with your suggestion and will fix the guide accordingly. 
I'm still new to cmake and DT is my current project to learn about it.